diff --git a/docker-compose/envs/common-frontend.env b/docker-compose/envs/common-frontend.env
index a3dcded763..bcaeb88eed 100644
--- a/docker-compose/envs/common-frontend.env
+++ b/docker-compose/envs/common-frontend.env
@@ -1,17 +1,19 @@
-NEXT_PUBLIC_API_HOST=localhost
-NEXT_PUBLIC_API_PROTOCOL=http
-NEXT_PUBLIC_STATS_API_HOST=http://localhost:8080
-NEXT_PUBLIC_NETWORK_NAME=Awesome chain
-NEXT_PUBLIC_NETWORK_SHORT_NAME=Awesome chain
-NEXT_PUBLIC_NETWORK_ID=5
-NEXT_PUBLIC_NETWORK_CURRENCY_NAME=Ether
-NEXT_PUBLIC_NETWORK_CURRENCY_SYMBOL=ETH
+NEXT_PUBLIC_API_HOST=devexplore.lestnet.org
+NEXT_PUBLIC_API_PROTOCOL=https
+NEXT_PUBLIC_STATS_API_HOST=https://devstats.lestnet.org
+NEXT_PUBLIC_NETWORK_NAME=Lestnet
+NEXT_PUBLIC_NETWORK_SHORT_NAME=Lestnet
+NEXT_PUBLIC_NETWORK_ID=21364
+NEXT_PUBLIC_NETWORK_CURRENCY_NAME=LesthetETH
+NEXT_PUBLIC_NETWORK_CURRENCY_SYMBOL=LETH
 NEXT_PUBLIC_NETWORK_CURRENCY_DECIMALS=18
 NEXT_PUBLIC_API_BASE_PATH=/
-NEXT_PUBLIC_APP_HOST=localhost
-NEXT_PUBLIC_APP_PROTOCOL=http
+NEXT_PUBLIC_APP_HOST=devexplore.lestnet.org
+NEXT_PUBLIC_APP_PROTOCOL=https
 NEXT_PUBLIC_HOMEPAGE_CHARTS=['daily_txs']
-NEXT_PUBLIC_VISUALIZE_API_HOST=http://localhost:8081
+NEXT_PUBLIC_VISUALIZE_API_HOST=https://devvisualizer.lestnet.org
 NEXT_PUBLIC_IS_TESTNET=true
-NEXT_PUBLIC_API_WEBSOCKET_PROTOCOL=ws
-NEXT_PUBLIC_API_SPEC_URL=https://raw.githubusercontent.com/blockscout/blockscout-api-v2-swagger/main/swagger.yaml
\ No newline at end of file
+NEXT_PUBLIC_API_WEBSOCKET_PROTOCOL=wss
+NEXT_PUBLIC_API_SPEC_URL=https://raw.githubusercontent.com/blockscout/blockscout-api-v2-swagger/main/swagger.yaml
+NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=5386985a4b5d406bc8c9fa78c6359ee7
+NEXT_PUBLIC_NETWORK_RPC_URL=https://devservice.lestnet.org
\ No newline at end of file
